### PR TITLE
docs: Kubernetes v2.7.0-1.18.6 documentation

### PR DESCRIPTION
**Pull request by tillt**
## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.
https://jira.d2iq.com/browse/D2IQ-70562

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/mesosphere/dcos/services/kubernetes/2.7.0-1.18.6/changelog/index.md [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+excerpt: Changelog for DC/OS Kubernetes
+---
+
+<!-- This source repo for this topic is https://github.com/mesosphere/dcos-kubernetes-cluster -->
+
+## Version 2.7.0-1.18.6
+
+### Changelog since 2.6.1-1.17.8
+
+## Improvements
```
Comment (sreis) [inline pages/mesosphere/dcos/services/kubernetes/2.7.0-1.18.6/changelog/index.md:15]:
```suggestion
#### Improvements
```

File: pages/mesosphere/dcos/services/kubernetes/2.7.0-1.18.6/release-notes/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+layout: layout.pug
+navigationTitle: Release Notes
+title: Release Notes
+menuWeight: 10
+excerpt: Release notes for DC/OS Kubernetes version 2.7.0-1.18.6
+---
+
+<!-- This source repo for this topic is https://github.com/mesosphere/dcos-kubernetes-cluster -->
+
+# Version 2.7.0-1.18.6
```
Comment (cmastr) [inline pages/mesosphere/dcos/services/kubernetes/2.7.0-1.18.6/release-notes/index.md:11]: Must have accurate release date before we merge.

File: pages/mesosphere/dcos/services/kubernetes/2.7.0-1.18.6/changelog/index.md [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+layout: layout.pug
+navigationTitle: Changelog
+title: Changelog
+menuWeight: 75
+excerpt: Changelog for DC/OS Kubernetes
+---
+
+<!-- This source repo for this topic is https://github.com/mesosphere/dcos-kubernetes-cluster -->
+
+## Version 2.7.0-1.18.6
```
Comment (cmastr) [inline pages/mesosphere/dcos/services/kubernetes/2.7.0-1.18.6/changelog/index.md:11]: Add release date.

File: pages/mesosphere/dcos/services/kubernetes/2.7.0-1.18.6/release-notes/index.md (hunk 2 above)
Comment (tillt) [reply pages/mesosphere/dcos/services/kubernetes/2.7.0-1.18.6/release-notes/index.md:11]: Talked to Jimmi and Simao about the release-date issue. 

There is a bit of a problem with that as the official release date is determined by the approval of the CNCF. In other words, until the CNCF had approved MKE’s new release candidate, we don’t have such date.

We should consider adding the date to the version (if really needed) as a final step --- after the docs itself got approved. Can we do this in another step for now?

**Pull request merged**